fingertip/plugins/script.py

```diff
@@ -0,0 +1,23 @@
+# Licensed under GNU General Public License v3 or later, see COPYING.
```

Comment:
Please make it into `fingertip.plugins.script.exec` by either renaming the function or adding an extra module and renaming a file. The reason is, I have plans for an auto-reexecuting thing and it feels logical to me to have them named `script.exec` and `script.debug`. Or maybe `run` and `debug`.